quickstart-templates/rhel-templats.json

```diff
@@ -0,0 +1,348 @@
+					"displayName": "Play builder name",
+					"name": "PLAY_BUILDER_NAME",
+					"value": "s2i-play",
+					"required": true
+				},
+				{
+					"description": "Git source URI for Play Framework builder",
+					"displayName": "Play builder source url",
+					"name": "PLAY_BUILDER_SOURCE_REPOSITORY_URL",
+					"value": "https://github.com/redhat-cop/containers-quickstarts.git",
+					"required": true
+				},
+				{
+					"description": "Play Framework builder source branch/tag reference.",
+					"displayName": "Play builder source branch/tag reference.",
+					"name": "PLAY_BUILDER_SOURCE_REPOSITORY_REF",
+					"value": "master",
+					"required": false
+				},
+				{
+					"description": "Path within Git project to build containing the Play Framework builder; empty for root project directory.",
+					"displayName": "Play builder context directory.",
+					"name": "PLAY_BUILDER_CONTEXT_DIR",
+					"value": "s2i-play",
+					"required": false
+				},
+				{
+					"description": "The name for the Play Framework application.",
+					"displayName": "Play application name",
+					"name": "PLAY_APPLICATION_NAME",
+					"value": "play-app",
+					"required": true
+				},
+				{
+					"description": "Git source URI for Play Framework application",
+					"displayName": "Play application source url",
+					"name": "PLAY_APPLICATION_SOURCE_REPOSITORY_URL",
+					"value": "https://github.com/playframework/play-ebean-example.git",
+					"required": true
+				},
+				{
+					"description": "Play Framework application source branch/tag reference.",
+					"displayName": "Play application source branch/tag reference.",
+					"name": "PLAY_APPLICATION_SOURCE_REPOSITORY_REF",
+					"value": "master",
+					"required": false
+				},
+				{
+					"description": "Path within Git project to build containing the Play Framework application; empty for root project directory.",
+					"displayName": "Play application context directory.",
+					"name": "PLAY_APPLICATION_CONTEXT_DIR",
+					"value": "",
+					"required": false
+				},
+				{
+					"name": "PLAY_APPLICATION_HOSTNAME",
+					"displayName": "Play application route hostname.",
+					"description": "Custom hostname for Play Framework service route.  Leave blank for default hostname, e.g.: \u003capplication-name\u003e-\u003cproject\u003e.\u003cdefault-domain-suffix\u003e"
+				},
+				{
+					"name": "PLAY_APPLICATION_JAVA_OPTS_EXT",
+					"displayName": "Play application Java extra parameters.",
+					"description": "Extra java parameters for the Play Framework application",
+					"value": "-Dplay.evolutions.db.default.autoApply=true",
+					"required": false
+				}
+			],
+			"objects": [
+				{
+					"kind": "ImageStream",
+					"apiVersion": "v1",
+					"metadata": {
+						"name": "rhel7",
+						"labels": {
+							"build": "${PLAY_BUILDER_NAME}"
+						}
+					},
+					"spec": {
+						"dockerImageRepository": "registry.access.redhat.com/rhel7/rhel"
```

Comment:
That only applies to Dockerfiles. ImageStreams still need full address since the API performs the resolution and not the docker daemon